nsacyber/HIRS
# [#847] Update HIRS Attestation CA and HIRS Attestation CA Portal Checkstyles.

### Description:
[Checkstyle](https://github.com/checkstyle/checkstyle/tree/master) is used for meeting best practices for java code styling. All classes in HIRS_AttestationCA and HIRS_AttestationCAPortal should be updated to pass checkstyle tests being run by gradle.

### How I Got Rid Of The Checkstyle Errors:
1. I ran the `./gradlew clean HIRS_AttestationCA:check --rerun-tasks` in the terminal window of my IDE. 
2. Because there were roughly 500-600 checkstyle errors, it was very difficult to scroll up and down the terminal window. So to make the process of reducing the errors easier, I had to write the output of the command into a another file.
3. I created a file called checkstyles_errors.txt using `touch checkstyles_errors.txt`.
4. Then I ran the gradle command and appended the output to the "end" of the new file, like so:
`./gradlew clean HIRS_AttestationCA:check --rerun-tasks  >  checkstyles_errors.txt  2>&1`
5. Since I wanted to keep an accurate count of how many checkstyle errors I encountered every time I ran this command, I would then run the following command: `grep -c "WARN" checkstyles_errors.txt`
6. These two commands allowed me to view the errors and keep track of how many errors were left.

### Test Instructions:
1. Follow steps 1 - 3 in this previous pull request: [issue #844 ](https://github.com/nsacyber/HIRS/pull/846) to confirm you can save and apply checkstyle recommended styling changes to your code in your favorite IDE.
7. Run the command below to confirm that all the checkstyle errors for the HIRS_AttestationCA module have been fixed (there were approximately 500 checkstyle errors in this module):
`./gradlew clean HIRS_AttestationCA:check --rerun-tasks`
8. Run the command below to confirm that that all the The HIRS_AttestationCAPortal module have been fixed (there were approximately 100 checkstyle errors in this module):
`./gradlew clean HIRS_AttestationCAPortal:check --rerun-tasks`
9.Run the command below to confirm that there are no issues building or running tests for the entire application:
`./gradlew clean build --rerun-tasks`
10. If you were able to run all three commands without running into any issues, you have successfully verified that the checkstyles guidelines have been applied to the these two modules successfully.

### Issues this PR addresses:
Closes issue #847
